apps/grow_bitcoin/sources/grow_bitcoin.move

```diff
@@ -416,6 +417,18 @@ module grow_bitcoin::grow_bitcoin {
         withdraw_token
     }
 
+    /// Harvest yield farming token from stake
+    public entry fun batch_harvest(signer:&signer, assets: vector<ObjectID>) {
+        let len = vector::length(&assets);
+        let i = 0;
+        while (i < len) {
+            let asset_id = *vector::borrow(&assets, i);
```

Comment:
We should use `object::borrow_mut_object` to check the ObjectID type and the ownership.